### PR TITLE
Set up testing environment for client(User)

### PR DESCRIPTION
**Pull request by Alex-Bellamy**
### User Story
```
As a development team
In order to be able to make sure our code is doing what it is supposed to do
we would like to be able to write and run automated tests
```
## Changes proposed in this pull request:
* adding testing environment
## What I have learned working on this feature:
* practice adding testing environment
## Packages/Gems added
- cypress
- jest with enzyme

### CODE REVIEW

**Review by Cma-B — APPROVED** [review]
good job !

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "baseUrl": "http://localhost:3001"
```
Comment (emtalen) [inline cypress.json]: Indentation here

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (emtalen) [inline cypress/fixtures/example.json]: Remove the examples

File: cypress/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```
Comment (oliverochman) [inline cypress/fixtures/example.json]: remove example fixture

**Review by emtalen — COMMENTED** [review]
If you want to use the script ```cy:run ``` with semaphore then you need to add that in

File: package.json [hunk 4]
```diff
@@ -14,7 +14,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "cy:run": "BROWSER=none yarn start & cypress run"
```
Comment (emtalen) [inline package.json]: well... Don't you need cy:open as well? 🤔

**Review by emtalen — APPROVED** [review]

**Pull request merged**